tests/simple-usage-spec.js

```diff
@@ -26,6 +26,33 @@ describe('rule engine usage', function() {
       });
   });
 
+  it('should match a simple regexp rule', function(done) {
+    rEngine.apply([{'view': { val: 'abcd' }}], [{'view': {val: /abcd/, should: true}}])
+      .then(function(res){
+        expect(res).toBe(true);
+      })
+      .then(done)
+      .catch(done);
+  });
+
+  it('should reject a simple regexp rule', function(done) {
+    rEngine.apply([{'view': { val: 'abcd' }}], [{'view': {val: /dcba/, should: true}}])
+      .then(function(res){
+        expect(res).not.toBe(true);
+      })
+      .then(done)
+      .catch(done);
+  });
+
+  it('should reject a regexp rule, specified as string', function(done) {
+    rEngine.apply([{'view': { val: 'abcd' }}], [{'view': {val: '/abcd/', should: true}}])
```

Comment:
In the current code state branch is covered by test and this test cover cases of true/false, but it is a good test to have anyway to test regression later on, will add it thanks